packages/perseus/src/widget-ai-utils/orderer/prompt-utils.ts

```diff
@@ -0,0 +1,28 @@
+import type {PerseusOrdererUserInput} from "../../validation.types";
+import type orderer from "../../widgets/orderer/orderer";
+import type React from "react";
+
+export type OrdererPromptJSON = {
+    type: "orderer";
+    options: {
+        options: string[];
+    };
+    userInput: {
+        values: ReadonlyArray<string>;
```

Comment:
**nit**: Can we use `ReadonlyArray<>` here?

Reply:
Do you mean to use `options: ReadonlyArray<string>` above?

Reply:
Yes, exactly. :)